algorithms/hhl/hhl.ipynb

```diff
@@ -0,0 +1,1367 @@
+{
```

Comment:
<p>Please write before this subsection that next we are going to treat two different examples: with an exact and approximated implementations for the Hamiltonian dynamics.</p><p> </p><p>Then, for each subsection, "exact Hamiltonian evolution" and "approximated Hamiltonian evolution", add a sentence that says what will be presented/used. For example, in the "exact Hamiltonian evolution" start with writing that here you will classically compute exp(iAt) and use the built-in <code>unitary</code> function. You can mention that it is not a scalable implementation, but it is good  for debugging and for treating small problems.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>